ui/app/models/pki/issuer.js

```diff
@@ -30,6 +30,10 @@ export default class PkiIssuerModel extends PkiCertificateBaseModel {
     return false;
   }
 
+  get issuerRef() {
+    return this.issuerName || this.issuerId;
```

Comment:
Is there any chance both of these would be null?

Reply:
nope!